platformio.ini

```diff
@@ -461,3 +461,19 @@ build_flags = ${common.build_flags} ${esp32s2.build_flags} -D WLED_RELEASE_NAME=
   -D HW_PIN_MISOSPI=9
 ;  -D STATUSLED=15
 lib_deps = ${esp32s2.lib_deps}
+
+
+[env:esp32dev_hub75]
```

Comment:
I will require this to be moved into sample file. I want platformio.ini clean and only used for default official builds.

Reply:
Done